packages/@lwc/engine-core/src/framework/hydration.ts

```diff
@@ -524,15 +524,15 @@ function areCompatibleNodes(client: Node, ssr: Node, vnode: VNode, renderer: Ren
             return false;
         }
 
-        return getProperty(client, 'nodeValue') === getProperty(ssr, 'nodeValue');
+        return getProperty(client, NODE_VALUE_PROP) === getProperty(ssr, NODE_VALUE_PROP);
     }
 
     if (getProperty(client, 'nodeType') === EnvNodeTypes.COMMENT) {
         if (!hasCorrectNodeType(vnode, ssr, EnvNodeTypes.COMMENT, renderer)) {
             return false;
         }
 
-        return getProperty(client, 'nodeValue') === getProperty(ssr, 'nodeValue');
+        return getProperty(client, NODE_VALUE_PROP) === getProperty(ssr, NODE_VALUE_PROP);
```

Comment:
The changes in this file seem unrelated to this PR?

Reply:
Yes, it is unrelated. I had made a change in this file but reverted that. Using the constant was part of that change, I left this part in because it was useful.